### PR TITLE
Change CONTRIBUTING.md to act as a 'table of contents' for the new contributor docs

### PR DESCRIPTION
**Pull request by milch**

### CODE REVIEW

**Review by KrauseFx — APPROVED** [review]
This is beautiful 🎷

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,188 +1,54 @@
 # Contributing to `fastlane`
```
Comment (mfurtak) [inline CONTRIBUTING.md]: backticks fastlane -> `_fastlane_`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,188 +1,54 @@
...
-- Then clone your new repo locally `git clone https://github.com/[my_user]/fastlane.git`
-- On your terminal, navigate to the project and run `git remote add upstream https://github.com/fastlane/fastlane` (or use the `git` URL if you use private key auth)
-- Create your feature branch (`git checkout -b my-new-feature`).
-- Commit your changes (`git commit -am 'Add some feature'`).
-- Push to the branch (`git push origin my-new-feature`).
-- Create a new Pull Request.
+Some people might also use the [fastlane tag on StackOverflow](https://stackoverflow.com/questions/tagged/fastlane), however we don’t actively monitor issues submitted there. 
 
-Before working on something, make sure to have pulled the latest changes. To pull the changes run
+## I want to contribute to fastlane
```
Comment (mfurtak) [inline CONTRIBUTING.md]: fastlane -> `_fastlane_`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,188 +1,54 @@
...
 
 In any case, **a closed issue is not necessarily the end of the story!** If more info becomes available after an issue is closed, it can be reopened for further consideration.
 
 One of the best ways we can keep fastlane an approachable, stable, and dependable tool is to be deliberate about how we choose to modify it. If we don't adopt your changes or new feature into fastlane, that doesn't mean it was bad work! It may be that the fastlane philosophy about how to accomplish a particular task doesn't align well with your approach. The best way to make sure that your time is well spent in contributing to fastlane is to **start your work** on a modification or new feature **by opening an issue to discuss the problem or shortcoming with the community**. The fastlane maintainers will do our best to give early feedback about whether a particular goal and approach is likely to be something we want to adopt!
```
Comment (mfurtak) [inline CONTRIBUTING.md]: fastlane -> `_fastlane_`

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,188 +1,54 @@
...
+## Above All, Thanks for Your Contributions
 
 Thank you for reading to the end, and for taking the time to contribute to the project! If you include the 🔑 emoji at the top of the body of your issue or pull request, we'll know that you've given this your full attention and are doing your best to help!
 
 ## License
-This project is licensed under the terms of the MIT license. See the LICENSE file.
+
+This project is licensed under the terms of the MIT license. See the [LICENSE][license] file.
 
 > This project and all fastlane tools are in no way affiliated with Apple Inc. This project is open source under the MIT license, which means you have full access to the source code and can modify it to fit your own needs. All fastlane tools run on your own computer or server, so your credentials or other sensitive information will never leave your own computer. You are responsible for how you use fastlane tools.
```
Comment (mfurtak) [inline CONTRIBUTING.md]: fastlane -> `_fastlane_`

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,188 +1,54 @@
...
-Before submitting a new issue, do the following:
+Please be aware that we don’t accept submissions for new actions at the moment. You can find more information about that [here][submit action].
 
-- Verify you're running the latest version by running `fastlane -v` and compare it with the [project page on GitHub](https://github.com/fastlane/fastlane/tree/master/fastlane).
-- Verify you have Xcode tools installed by running `xcode-select --install`.
-- Make sure to read through the [README](https://github.com/fastlane/fastlane/tree/master/fastlane) of the project.
+## I want to help work on _fastlane_ by reviewing issues and PRs
 
-When submitting a new issue, please provide the following information:
+We are always happy to receive help in that regard by the community! Feel free to read our document on how to [respond to issues and PRs][responding to prs] and also check out how to become a [core contributor][core contributor].
```
Comment (mfurtak) [inline CONTRIBUTING.md]: Suggestion:

"We are always happy to receive help in that regard by the community!" -> 

"Thanks! We would really appreciate the help!"

**Comment by mfurtak** [comment]
A great way to tie it all together 🎀  I left some small suggestions and formatting notes. 👍

**Comment by milch** [comment]
@mfurtak Thanks, I addressed your review!

**Comment by mfurtak** [comment]
Looks great, :ship: it!

**Pull request merged**